completion/available/hub.completion.bash

```diff
@@ -23,12 +23,12 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 # If there is no git tab completion, but we have the _completion loader try to load it
-if ! declare -F _git > /dev/null && declare -F _completion_loader > /dev/null; then
+if ! _command_exists _git && _command_exists _completion_loader; then
```

Comment:
Since this was specifically testing for being a function, you might want to use [`_is_function`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L167)